NYPL/MyLibraryNYCApp
# TGR 112 implement docker compose

Keeping this work on a (sideline) epic feature branch for now, because even setting up the application to this point required upgrading Node. We'll plan to validate and release all upgrade changes at once. Until then, the latest qa branch version should regularly be pulled into `tiger-team-feature`.

## JIRA tickets:
- [MyLibraryNYC Investigation(s)](https://newyorkpubliclibrary.atlassian.net/browse/TGR-112)

## Things Done:
- Upgraded Node to 20.x
- Set up docker compose for local development

## How to Test:
- Create the docker images with `docker-compose build --no-cache`
- Bring up the docker environment with `docker-compose up`
- Enter a running webapp container with `docker-compose exec -it webapp bash`
- `npm run test`
- The application should operate as expected in the development environment.
